src/Requests/VerifyChallengeRequest.php

```diff
@@ -61,15 +61,24 @@ public function getMessage()
      */
     public function handleResponse($data)
     {
-        // Retrieve the specific request data
-        $requestData = current($data->getReturnsArray());
+        // DEBUG - dump response object
+        //var_dump($data);
+        // DEBUG - check status code of response for error handling, but error can return both 1 and 2 - not sure why?
```

Comment:
Both mean the request succeeded, the difference is that a `2` response also contains a new endpoint / API-URL (*RPC URL*).

See the enum in [ResponseEnvelope](https://github.com/AeonLucid/POGOProtos/blob/master/src/POGOProtos/Networking/Envelopes/ResponseEnvelope.proto):
``` protobuf
OK = 1; // valid response with no api url
OK_RPC_URL_IN_RESPONSE = 2; // the response envelope has api_url set and this response is valid
```